doc/architecture/decisions/0002-store-etl-output-in-azure.md

```diff
@@ -0,0 +1,19 @@
+# 2. Store ETL output in Azure
```

Comment:
Reading this I don't feel like I know why Azure was chosen over another hosted file server. It would be good to know that as I'm sure this will be a question asked in the future.